modules/configuring-firewall.adoc

```diff
@@ -82,15 +82,57 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 |Cloud |URL | Port |Function
 
 |Alibaba
-|*.aliyuncs.com
+|`*.aliyuncs.com`
 |443, 80
 |Required to access Alibaba Cloud services and resources. Review the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/sdk/endpoints/endpoints_config.go?spm=a2c4g.11186623.0.0.47875873ciGnC8&file=endpoints_config.go[Alibaba endpoints_config.go file] to determine the exact endpoints to allow for the regions that you use.
 
-|AWS
+.11+|AWS
 |`*.amazonaws.com`
+
+Alternatively, if you choose to not use a wildcard for Amazon Web Services (AWS) APIs, you must allowlist the following URLs:
 |443, 80
 |Required to access AWS services and resources. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
 
+|`ec2.amazonaws.com`
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|`events.amazonaws.com`
+|443
+|Used to install and manage clusters in an AWS environment.
+
+|`iam.amazonaws.com` [^1^]
```

Comment:
@mjpytlak I think we don't need to add a footnote for IAM endpoint, aligning with the [existing document](https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html-single/prepare_your_environment/index#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs) is better.

Reply:
Removed.